### ISSUE TITLE
New DSN triggers "Invalid Sentry DSN" error

### ISSUE THREAD

**Issue by birkof** [getsentry/sentry-symfony#142]
On newer versions of Sentry's SDK the secret is no longer required in DSN.

> vendor/sentry/sentry/lib/Raven/Client.php[511]
```
if (empty($netloc) || empty($project) || empty($username) || empty($password)) {
    throw new InvalidArgumentException('Invalid Sentry DSN: ' . $dsn);
}
```

The last condition should be removed...

**Comment by stayallive** [comment]
Being addressed in #615

**Comment by stayallive** [comment]
You can `composer update` to `1.9.1` of `sentry/sentry-php` (automatically pulled in with the latest `sentry/sentry-symfony`) and that should be happy to accept the public DSN 👍
